pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -222,11 +229,29 @@ func (w *nfdTopologyUpdater) Stop() {
 }
 
 func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneList, scanResponse resourcemonitor.ScanResponse, readKubeletConfig bool) error {
+
+	if len(w.ownerRefs) == 0 {
+		ns, err := w.k8sClient.CoreV1().Namespaces().Get(context.TODO(), w.kubernetesNamespace, metav1.GetOptions{})
+		if err != nil {
+			klog.ErrorS(err, "Cannot get NodeResourceTopology owner reference")
+		} else {
+			w.ownerRefs = []metav1.OwnerReference{
+				{
+					APIVersion: "v1",
+					Kind:       "Namespace",
```

Comment:
Maybe something like this to avoid random strings?
```suggestion
					APIVersion: ns.ApiVersion,
					Kind:       ns.Kind,
```

Reply:
@PiotrProkop
Thanks, updated.